.github/pull_request_template.md

```diff
@@ -1,3 +1,7 @@
 ## Summary & Motivation
 
 ## How I Tested These Changes
+
+## Did you update the changeset?
+
+You should add file to describe the change from this pr that looks like `.changeset/<changename>.md` ([example](https://github.com/tkhq/sdk/blob/b409cd06790f011bf939adcf0755499b8e7497ae/.changeset/extra-http-exports.md?plain=1#L1)). Note that at the top it includes the package name along with the type of [semver change (major.minor.patch).](https://semver.org/). These changes will be used at release time to determine what packages to publish and how to bump their version. For more context see [this comment](https://github.com/tkhq/sdk/pull/67#issuecomment-1568838440).
```

Comment:
You don't need to manually create files: `pnpm run changeset` does it for you and knows which packages were changed automatically!